components/src/molecules/ParametersTable/index.tsx

```diff
@@ -7,8 +7,9 @@ import {
 } from '@opentrons/shared-data'
 import { BORDERS, COLORS } from '../../helix-design-system'
 import { SPACING, TYPOGRAPHY } from '../../ui-style-constants/index'
+import { TYPOGRAPHY as HELIX_TYPOGRAPHY } from '../../helix-design-system/product'
```

Comment:
nit

probably using legacy would be consistent

```suggestion
import { SPACING, TYPOGRAPHY as LEGACY_TYPOGRAPHY } from '../../ui-style-constants/index'
import { TYPOGRAPHY } from '../../helix-design-system/product'
```

Reply:
better idea